std/uni.d

```diff
@@ -622,7 +622,7 @@ import std.range.primitives; // back, ElementEncodingType, ElementType, empty,
     // save
 import std.traits; // isConvertibleToString, isIntegral, isSomeChar,
     // isSomeString, Unqual
-
+import std.exception; // enforce
```

Comment:
Oh for **ck sake... I'll change this one but the other will stay as is for now, I don't want to mix concerns in a single pull request that supposed to be refactoring.